DataTool/ToolLogic/List/ListHeroes.cs

```diff
@@ -34,7 +34,7 @@ public class HeroInfo {
                 GUID = guid;
                 Name = name;
                 Description = description;
-                Color = color;
+                Color = color.ToHex();
```

Comment:
Keep this as the raw color

Reply:
Sidenote: this class isn't even used here

Reply:
Only turn the colours to hex when you are displaying them to the user

Reply:
The struct is quite useful

Reply:
oh lol didnt test the json, i assumed it was used. The only problem with keeping as `teColorRGBA` means the JSON thing can't stringify it, it would need a custom JSONConverter like the GUIDConverter?

Reply:
unless we just store two, ColorString and Color

Reply:
probs easier tbh

Reply:
we can make teColorRGB(A) a json object. it'll just write the R,G,B values